tests/integration_tests/test_guard.py

```diff
@@ -1243,7 +1243,7 @@ def test_guard_from_pydantic_with_mock_hf_pipeline():
 
     pipe = make_mock_pipeline()
     guard = Guard()
-    _ = guard(pipe, prompt="Don't care about the output.  Just don't crash.")
```

Comment:
The context length with the upgraded version was 20 characters instead of 24. :/ I could update the model, but then it would be a whole can of worms.  Easier to trim the prompt.